core/src/main/groovy/noe/ews/workspace/WorkspaceEws.groovy

```diff
@@ -44,7 +42,7 @@ class WorkspaceEws extends WorkspaceAbstract {
    */
   def prepare() {
     workspaceHttpd?.prepare()
-    if (platform.isWindows()) {
+    if (platform.isWindows() && workspaceHttpd != null) {
```

Comment:
```suggestion
    if (platform.isWindows() && workspaceHttpd) {
```
would be same, imho.

Reply:
I think so...

Reply:
`&& workspaceHttpd` will indeed work in the same manner.